torch/csrc/jit/passes/quantization.cpp

```diff
@@ -23,7 +72,30 @@ void InsertObserverNodes(std::shared_ptr<Graph>& graph) {
 }
 
 void InsertFakeQuantNodes(std::shared_ptr<Graph>& graph) {
-  throw std::runtime_error("Pass not implemented yet!");
+  for (const auto& n : graph->nodes()) {
```

Comment:
Please also add a code for visiting subblocks. It's very straightforward to add and there is no reason for restricting to a single block.